README.md

```diff
@@ -1 +1,234 @@
-# To-Do-List
\ No newline at end of file
+# To Do List
+HTML, CSS and JavaScript Awesome Books Project
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [Tech Stack](#tech-stack)
+- [Key Features](#key-features)
```

Comment:
- [ ] In the README file please add steps to set up and run the project, It will make the README file more professional. Example: `npm install` `npm run start`.